packages/at_login_flutter/lib/services/notification_service.dart

```diff
@@ -27,15 +27,16 @@ class NotificationService {
 
   setOnNotificationClick(Function onNotificationClick) async {
     await _notificationsPlugin.initialize(initializationSettings,
-        onSelectNotification: (String? payload) async {
-      onNotificationClick(payload);
-    });
+        onDidReceiveNotificationResponse: (NotificationResponse payload) {
+          onNotificationClick(payload);
+        });
   }
```

Comment:
The _notificationsPlugin.initialize API changed, I'm not sure if this is correct, I will be investigating this one for breaking changes.